Added login and register form

ahmedjasarevic opened the pull request:
Homepage navbar is now fixed. Login / Register / Logout are now aligned.
I have done register and login form using bootstrap.When you click register it will make new account.Login will login you to your account.

mehmedkadric commented on aMovieDatabase/core/templates/login.html:
```diff
@@ -1,24 +1,40 @@
+      <link rel="stylesheet" type="text/css" href="{%static 'style.css' %}">
+</head>
+<body>
+<div class="login-form">
```
Login and register content should be moved to accounts app.

ahmedjasarevic replied:
Login and register are now moved to accounts app

mehmedkadric commented on aMovieDatabase/core/templates/login.html:
```diff
@@ -8,7 +8,8 @@
                {% csrf_token %}
-        <h2 class="text-center">Log in</h2>
+            <h2>Login</h2>
+		<p class="hint-text">Welcome back! Log in to access your account.</p>
```
In login form, when I enter a username and move cursor to another input field, username text color turns into gray.

ahmedjasarevic replied:
Now when you move cursor it stays black

ahmedjasarevic commented:
I have moved navbar from base.html to includes/navbar.html to make code clear . For login and register I have added toast messages, you can find them in messages.html. For succes login, you will see username displayed at navbar.

mehmedkadric reviewed (approved):
LGTM! 🚀
